Realm/Realm/Migration.cs

```diff
@@ -83,11 +86,46 @@ internal bool Execute(Realm oldRealm, Realm newRealm)
 
                 NewRealm.Dispose();
                 NewRealm = null;
+
+                _migrationSchema = IntPtr.Zero;
             }
 
             return true;
         }
 
+        /// <summary>
+        /// Removes a type during a migration. All the data associated with the type, as well as its schema, will be removed from <see cref="Realm"/>.
```

Comment:
Do we need to differentiate between the (general) schema and the object schema to not confuse anyone here?

Reply:
I think it's clear enough with the "its", but maybe it's not?

Reply:
Clear enough for me because I know how Realm works and what the difference between a schema and an object schema is. Then again, how important is this for a user? They just want to use the framework and not understand too much about the internals I suppose. I guess it's fine both ways. ;)